File: Assignment__33/Question3.c
#include<stdio.h>
#include<stdbool.h>

bool checkDigit(char *str)
{
    while(*str != '\0')
    {
        if((*str >= 48 && *str <= 57))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

int main()
{
    char Arr[50] = "\0";
    bool bRet = false;

    printf("Enter the String : \n");
    scanf("%[^\n]s",Arr);

    bRet = checkDigit(Arr);

    if(bRet == true)
    {
        printf("It is a Digit.");
    }
    else
    {
        printf("It is not a Digit");
    }

    return 0;
}

File: Assignment__33/Question3.java
import java.util.*;

class Digit
{
    public boolean Check(char chh)
    {
        if((chh >= 48 && chh <= 57))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
class Question3
{
    public static void main(String A[])
    {
        boolean bRet = false;

        Scanner scanobj = new Scanner(System.in);
        System.out.println("Enter the input : ");

        char ch = scanobj.next().charAt(0);               //Need to remember

        Digit aobj = new Digit();

        bRet = aobj.Check(ch);

        if(bRet == true)
        {
            System.out.println("It is a Digit");
        }
        else
        {
            System.out.println("It is not a Digit");
        }

    }
}